ironfish-cli/src/commands/wallet/import.ts

```diff
@@ -34,24 +47,103 @@ export class ImportCommand extends IronfishCommand {
     },
   ]
 
+  async importLedger(): Promise<RpcAccountImport> {
+    const PATH = "m/44'/1338'/0"
```

Comment:
Won't this path change depending on how many accounts are present for a ledger device? For example will last bit be `../1` for a second account?